.eslintrc.js

```diff
@@ -52,7 +52,11 @@ module.exports = {
             }
         ],
         "node/no-unpublished-bin": "error",
-        "node/no-unpublished-require": "error",
+        "node/no-unpublished-require": ["error", {
```

Comment:
Where should I look? This was failing on my local and in Travis. When I did some looking this was the suggested answer from google. I dug a little more but found no other path. Just wondering what suggestions you may have.

Reply:
That's because you added `assets-webpack-plugin` as a dev dependency.

Since people installing Encore in their project won't get our dev dependencies the linter is telling you that they also won't have the `assets-webpack-plugin`.

The dev dependencies in Encore are used for optional things only (for instance if you use it with vue.js you'll have to add the loader manually), in this case we check that the dependency is present before requiring it (see `notifier.js` for instance).

For this plugin it makes more sense to add it to normal dependencies, so people will automatically get it when they install Encore.

Reply:
Okay I will move it. Thank you!